### PR TITLE
Update default etcd server to 3.2.24 for kubernetes 1.12

### PR DESCRIPTION
**Pull request by timothysc**
**What this PR does / why we need it**:
Update default etcd server to 3.2.24 for kubernetes 1.12

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
xref #68147 

**Special notes for your reviewer**:
NONE

**Release note**:

```
Update default etcd server to 3.2.24 for kubernetes 1.12
```

/assign @wojtek-t @jpbetz @dims 
/cc @kubernetes/sig-cluster-lifecycle-pr-reviews @gyuho

### CODE REVIEW

File: test/e2e/framework/nodes_util.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func etcdUpgradeGCE(target_storage, target_version string) error {
 		os.Environ(),
 		"TEST_ETCD_VERSION="+target_version,
 		"STORAGE_BACKEND="+target_storage,
-		"TEST_ETCD_IMAGE=3.2.18-0")
+		"TEST_ETCD_IMAGE=3.2.24-0")
```
Comment (wojtek-t) [inline test/e2e/framework/nodes_util.go]: Please also modify test/kubemark/start-kubemark.sh

**Review by luxas — APPROVED** [review]
LGTM

**Comment by dims** [comment]
i cross checked the changes in this PR with what i had from earlier and it looks good except for the one spot that @wojtek-t mentioned already in `test/kubemark/start-kubemark.sh`

```
# Etcd related variables.
ETCD_IMAGE="${ETCD_IMAGE:-3.2.18-0}"
ETCD_VERSION="${ETCD_VERSION:-}"
```

File: test/e2e/framework/nodes_util.go (hunk 1 above)
Comment (timothysc) [reply test/e2e/framework/nodes_util.go]: Done, plus I found one other test that needed an update too.

**Comment by timothysc** [comment]
@wojtek-t PTAL.

**Comment by dims** [comment]
+1 from me

**Pull request merged**